grants/milestone-delivery-guidelines.md

```diff
@@ -0,0 +1,29 @@
+# Milestone Deliverables Guidelines
+
+We kindly ask you to consider the following guidelines when you submit a milestone for evaluation. Please submit your milestones by email to grants@web3.foundation.
+
+The submission should contain the following information: 
+ * **Name of the grant project**
+ * **Link to the open-source code/delivery** 
+ * **Basic [testing guide](#testing-guide)** 
+ * **A list of the [milestone deliverables](#milestone-deliverables)**
+ * **Any [additional information](#additional-information)**
+
+## Testing Guide
+
+Please provide documentation on how to install, compile, run, and/or test the delivery. Documentation must be publicly available, such as a readme file or similar. This will make it easier for community to use or adapt this project. For installation, we prefer docker images to avoid problems with versions and dependencies.
+
+## Milestone Deliverables
+
+Please provide a list of milestone deliverables. This list should closely reflect the list of deliverables shown in Annex 1 of the grant contract. Each item in the list should include a link to the deliverable itself. If necessary, please highlight anything that deviates from the contract and include further information that you think is relevant to deliverable.
```

Comment:
" include further information that you think is relevant to **the** deliverable."  (I must have missed this)

I'd be inclined to move "Please ensure the repo has the correct open-source license." above the table, But a minor point.


Otherwise good to merge I think.